Changes

```diff
@@ -1,5 +1,6 @@
 
 10.29  2023-11-28
+  - Improved admin ui to show the lock ID on the locks page.
```

Comment:
Don't write your own Changes entries please, those are done by the maintainers.

Reply:
Instead it needs to be mentioned in the documentation, like the other fields.

Reply:
you are correct, I have corrected this.